packages/palette/src/elements/Radio/Radio.tsx

```diff
@@ -39,6 +41,7 @@ export const Radio: React.FC<RadioProps> = ({
   label,
   selected,
   value,
+  size = "small",
```

Comment:
I'd recommend using the same size nomenclature we use elsewhere: `sm`, `md`, `lg`